src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -488,3 +488,49 @@ def test_bz1927195(multihost, backupsssdconf, proxy_sleep):
                                         "queuing request' "
                                         "/var/log/sssd/sssd_example1.log")
         assert result.stdout_text == '1\n'
+
+    @staticmethod
+    def test_netgroup(multihost, backupsssdconf):
+        """
+        :title: Lookup alias and original netgroups
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=804103
+        :id: b35a6a2e-0ce5-11ed-b2dc-845cf3eff344
+        :steps:
+          1. Check alias and original netgroups.
+          2. Check if returning non-existant netgroups.
+          3. Check while case sensitive is false lookup netgroups.
+        :expectedresults:
+          1. Should succeed
+          2. Should not succeed
+          3. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'debug_level': '0xFFF0',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'ldap',
+                         'proxy_pam_target': 'sssdproxyldap'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        # Lookup alias and original netgroups
+        execute_cmd(multihost, "getent netgroup NetGroup_CS1 | "
```

Comment:
Hi,

where is this netgroup added to the LDAP server?

bye,
Sumit

Reply:
This is part of class fixture :

@pytest.mark.usefixtures('setup_sssd_krb',
                         'create_posix_usersgroups',
                         'sssdproxyldap',
                         'install_nslcd',
                         'create_user_with_cn',
                         'netgroups')

Reply:
Hi,

if I see it correctly `netgroups` from `src/tests/multihost/alltests/conftest.py` only creates `netgroup_1` ... `netgroup_10` not the one you are using.

bye,
Sumit

Reply:
Sorry, added that part to same fixture